client/src/components/ConfirmEmail.js

```diff
@@ -40,6 +40,7 @@ const ConfirmEmail = props => {
       ) : confirmResult.success ? (
         <Redirect to={`/login/${email}`} />
       ) : emailSent ? (
+        // TODO: CHECK ON THIS CODE - How do we test this to see this?? -- Claire

```

Comment:
You would test this by setting up an invalid or expired token, either by waiting an hour after registration for the token to expire, or by manually deleting the token from the database before clicking on the link in the email.  Then, when you get to this page, the result should be an expired or invalid token, an the user should be instructed to push the button to generate a fresh token and send another confirmation email, which should cause this code to execute.